spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/UnsecuredProxyRepository.java

```diff
@@ -0,0 +1,45 @@
+package eu.coatrack.admin.model.repository;
+
+/*-
+ * #%L
+ * coatrack-api
+ * %%
+ * Copyright (C) 2013 - 2020 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import eu.coatrack.api.Proxy;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.rest.core.annotation.RestResource;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PostFilter;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * These unsecured methods are only to be used within an authenticate() method. The lack of
+ * method security prevents StackOverflow exceptions when these methods are called within authenticate().
+ * @author Christoph Baier
+ */
+@RepositoryRestResource(collectionResourceRel = "proxies", path = "proxies")
```

Comment:
It could just remove that annotation, it is not required for functionality. I am not sure why I added it at all. And referring to your second point, I think that @BrunoSilvaATB mentioned that he added a solution for that in another branch. To be discussed.

Reply:
I assume that `@RepositoryRestResource` was simply a copy from the other repository classes, which are exposed as REST endpoints. If it is not required here, then there should be no REST endpoint.

I assume that another annotation will be required as a replacement to tell spring that this is a "repository to be implemented".

Reply:
After looking at the code, it seems like it also works without any annotation. :+1: